bokehjs/test/integration/glyphs/markers.ts

```diff
@@ -112,4 +113,39 @@ describe("Marker glyph", () => {
 
     await display(row([p0, p1, p2]))
   })
+
+  it("should support hatch", async () => {
+    const hatch_patterns: HatchPattern[] = [
+      "dot", "ring", "horizontal_line", "vertical_line", "cross", "horizontal_dash",
+      "vertical_dash", "spiral", "right_diagonal_line", "left_diagonal_line", "diagonal_cross",
+      "right_diagonal_dash", "left_diagonal_dash", "horizontal_wave", "vertical_wave",
+      "criss_cross", "dot", "ring", "horizontal_line", "vertical_line", "cross",
+    ]
+
+    const markers: MarkerType[] = [
+      "circle", "circle_cross", "circle_dot", "circle_x", "circle_y", "diamond", "diamond_cross",
+      "diamond_dot", "hex", "hex_dot", "inverted_triangle", "square", "square_cross", "square_dot",
+      "square_pin", "square_x", "star", "star_dot", "triangle", "triangle_dot", "triangle_pin",
+    ]
+
+    function make_plot(output_backend: OutputBackend) {
+      const p = fig([300, 300],
+        {output_backend, title: output_backend, x_range: [-0.5, 4.5], y_range: [-0.6, 4.6]})
```

Comment:
To match the existing code style, this:
```ts
const p = fig([300, 300], {
  output_backend, title: output_backend,
  x_range: [-0.5, 4.5], y_range: [-0.6, 4.6],
})
```
or this
```ts
const p = fig([300, 300], {
  output_backend, title: output_backend,  x_range: [-0.5, 4.5], y_range: [-0.6, 4.6],
})
```
is preferred.

Reply:
Will do.